src/zenml/models/v2/base/internal.py

```diff
@@ -15,10 +15,13 @@
 
 from typing import Type, TypeVar
 
+from zenml.logger import get_logger
 from zenml.models.v2.base.base import BaseRequest
 
 T = TypeVar("T", bound="BaseRequest")
 
+logger = get_logger(__name__)
```

Comment:
why was this added? I guess for debugging? Maybe remove?

Reply:
Yeah, will remove this

Reply:
Removed